### PR TITLE
Move all index to atomic files.

### PR DESCRIPTION
**Pull request by RagnarW**
Index ids will for the time being always be an empty set in prepare
store copy request.

### CODE REVIEW

File: enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/storecopy/CatchupServerIT.java [hunk 1]
```diff
@@ -277,10 +277,9 @@ private void listOfDownloadedFilesMatchesServer( NeoStoreDataSource neoStoreData
         assertThat( givenFile, containsInAnyOrder( expectedStoreFiles.toArray( new String[givenFile.size()] ) ) );
     }
 
-    private void assertIndexIdsMatch( PrimitiveLongSet indexIds, NeoStoreDataSource neoStoreDataSource )
+    private void assertIndexIdsAreEmpty( PrimitiveLongSet indexIds )
     {
-        PrimitiveLongSet expectedIndexIds = getExpectedIndexIds( neoStoreDataSource );
-        assertThat( expectedIndexIds, equalTo( indexIds ) );
+        assertTrue( indexIds.isEmpty() );
```
Comment (andrewkerr9000) [inline enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/storecopy/CatchupServerIT.java]: `assertThat( indexIds, Matchers.isEmpty() );` would give better output on failure. May as well be inlined too.

File: enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/storecopy/PrepareStoreCopyFilesTest.java [hunk 2]
```diff
@@ -122,21 +123,21 @@ public void shouldReturnExpectedListOfFileNamesForEachType() throws Exception
...
 
         assertEquals( 0, indexIds.size() );
     }
 
     @Test
-    public void shouldReturnExpectedDescriptors()
+    public void shouldReturnEmptySetOfIds()
     {
         PrimitiveLongSet expectedIndexIds = Primitive.longSet();
         expectedIndexIds.add( 42 );
```
Comment (andrewkerr9000) [inline enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/storecopy/PrepareStoreCopyFilesTest.java]: Unused `expectedIndexIds`

**Review by andrewkerr9000 — COMMENTED** [review]
I'm not familiar with this area, so I can't really judge the consequences of this change.

File: enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/storecopy/CatchupServerIT.java (hunk 1 above)
Comment (RagnarW) [reply enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/storecopy/CatchupServerIT.java]: This is primitive collection so can't do that. But I'll inline and add a custom message

**Review by chrisvest — APPROVED** [review]

File: enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/storecopy/CatchupServerIT.java (hunk 1 above)
Comment (andrewkerr9000) [reply enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/storecopy/CatchupServerIT.java]: You could if you wrote a Matcher :)

**Pull request merged**